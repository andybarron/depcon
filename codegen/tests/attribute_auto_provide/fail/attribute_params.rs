use depcon::*;

trait Interface {}
struct Implementation;

#[auto_provide(bad)]
impl Interface for Implementation {}

fn main() {}
