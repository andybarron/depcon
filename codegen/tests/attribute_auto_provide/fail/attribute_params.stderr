error: custom attribute panicked
 --> tests/attribute_auto_provide/fail/attribute_params.rs:6:1
  |
6 | #[auto_provide(bad)]
  | ^^^^^^^^^^^^^^^^^^^^
  |
  = help: message: attribute does not accept parameters
